inference-engine/include/ie_parameter.hpp

```diff
@@ -254,6 +255,21 @@ class INFERENCE_ENGINE_API_CLASS(Parameter) {
         return !(*this == rhs);
     }
 
+    /**
+     * @brief Prints underlying object to the given output stream.
+     * Uses operator<< if it is defined, leaves stream unchanged otherwise.
+     * In case of empty parameter or nullptr stream immediately returns.
+     *
+     * @param object Object to be printed to the given output stream.
+     * @param stream Output stream object will be printed to.
+     */
+    friend void PrintTo(const Parameter& object, std::ostream* stream) {
```

Comment:
BTW, why not standard `operator <<`

Reply:
For explanations see commit message. Thanks.

Reply:
It is better to implement `operator <<` here and `PrintTo` function that uses the operator  in some test header as it is `GTest` specific API.

Reply:
Please see commit message for explanations why `std::operator<<` is not implemented.